data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,2721 @@
+			`	"Worry not. Respect is only to be earned once. From me, at least. Do know there are other captains eager to challenge you, and most are stronger than me. Or, at least, they have more ships..."`
+			label end
+			action
+				log `Defeated a young Unfettered captain in a duel.`
+				set "unfettered: beaten the captain"
+				payment 100000
+				"reputation: Hai (Unfettered)" += 30
+			`	"Hopefully we see each other again, sometime." He and his crew seem quite busy with the repairs. "I have transferred a small amount of <payment> to you to repair any damage I may have done to your ship. I will be landing on the surface soon to think about something other than this humiliating defeat I have just suffered. I do not need nor want your assistance in repairing damage, for that is a task best suited for the vanquished."`
+			`	"If you are feeling up to the challenge there are other captains that would be willing to duel you, if you do not have a fleet too big of course. None accept to fight more than 2 heavy warhips, some even prefer just 1, like me. Indeed we wish to fight and test your strength and exotic technologies or strategies, not to get humiliated."`
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on visit
+		dialog "You have not yet defeated the Unfettered captain. If he did not show up it is probably because your fleet is too large."
+	to complete
+		has "unfettered: beaten the captain"
+	on complete
+		clear "unfettered: beaten the captain"
+
+mission "Unfettered: Trust Broken"
+	landing
+	to offer
+		has "Unfettered: Prove Your Strength: active"
+		"ships: Transport" * 6 + "ships: Light Freighter" * 3 + "ships: Heavy Freighter" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 4 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 <= 12
+	to fail
+		"ships: Transport" * 6 + "ships: Light Freighter" * 3 + "ships: Heavy Freighter" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 4 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 > 12
+	on fail
+		dialog "You have broken the trust of the Unfettered captain by trying to come to a duel with too many ships. He will not show up again."
+		fail "Unfettered: Prove Your Strength"
+	to complete
+		has "unfettered: beaten the captain"
+
+
+
+mission "Unfettered: Prove Your Strength 2"
+	name `Recover the jump drives`
+	description `Go to the pirate world of <stopovers> and recover jump drives from a vault.`
+	source "Darkcloak"
+	stopover "Freedom"
+	passengers 1
+	to offer
+		has "Unfettered: Prove Your Strength: done"
+		has "Unfettered Jump Drive 3: offered"
+	on offer
+		conversation
+			`As you pass what looks like a bar in the spaceport, you're greeted by the same Hai captain that challenged you earlier. "So we meet again! Next time we fight it will take a lot more if you want to take me down! Well... once I have my ship back, anyway. But that's not why I am here," he says, apparently happy to see you. "I have a job for you!"`
```

Comment:
Isnt clipper a kind of ship?
I guess this is well known pirate expression

Reply:
It is also the name of a specific ship in the game, yes.

Mostly was looking to avoid repetition of "ship" or "vessel," but I did want to kind of lean into that confusion though, as it's also an approximately appropriate description of the type of ship the sea scorpion is.

If others think it's -too- confusing, just would need to swap it with something else.